[terra-arrange] add reflow support for 400% zoom (#3846)
cerner/terra-core: packages/terra-arrange/CHANGELOG.md

## Unreleased

> * Fixed
>   * Fixed issue terra arrange support resizing to 400% and viewport is resized to 320x256 px without loss of information or function.

Reviewer: ```suggestion
* Fixed
  * Fixed reflow issue when zoomed to 400% and the viewport is resized to 320x256 px.
```

Author: i have made changes as per suggestion.
